src/gameplayv2/resistance/components/SeeSpiesInfoComponent.ts

```diff
@@ -0,0 +1,36 @@
+export class SeeSpiesInfoComponent {
```

Comment:
Probably don't need `Info` in this class name. `SeeSpiesComponent` is probably adequate :)